server/protocols/webadmin-cli/IMPLEMENT_WEBADMINCLI.md

```diff
@@ -0,0 +1,48 @@
+# Architecture Decision Record for new James CLI
+
+## Why implement a CLI based on webadmin API
+
+James servers offer a command-line interface in order to interact with the server. However, it relies on the JMX protocol, which is known to be insecure.
+
+Webadmin APIs use HTTP protocol, which is more secure than JMS protocol to interact with James servers.
+
+Webadmin command-line interface is an upcoming replacement for the outdated, security-vulnerable JMX command-line interface.  
+
+## Context
+We decided some points about new James CLI architecture:
+
+* What libraries will we use? 
+
+  * http client: Feign library
```

Comment:
Why do we use feign? (because we use it as an http client in other parts of james).